src/hostname/hostnamectl.c

```diff
@@ -150,6 +153,7 @@ static int show_all_names(sd_bus *bus, sd_bus_error *error) {
                 { "OperatingSystemPrettyName", "s", NULL, offsetof(StatusInfo, os_pretty_name)  },
                 { "OperatingSystemCPEName",    "s", NULL, offsetof(StatusInfo, os_cpe_name)     },
                 { "HomeURL",                   "s", NULL, offsetof(StatusInfo, home_url)        },
+                { "Model",                     "s", NULL, offsetof(StatusInfo, model)           },
```

Comment:
shouldn't there at least be "Vendor" too here?

Reply:
"Vendor" is ambiguous — it may refer to the OS vendor, and other things. Maybe "HardwareModel" ?

Reply:
`HardwareModel i s more like a family string.
`SystemVendor` perhaps?  That at least disambinguates and matches what would be sourced from `/sys/class/dmi`

Reply:
I think we should include a prefix like "Hardware" here, too. HardwareVendor and HardwareModel sounds good to me?